--- daemon/castcast/amazon_drm.py
+++ daemon/castcast/amazon_drm.py
@@ -95,9 +95,9 @@
                 "streamingTechnologies": {
                     "DASH": {
                         "bitrateAdaptations": ["CVBR"],
-                        "codecs": ["H265", "H264"],
+                        "codecs": ["H265"],
                         "drmType": "Widevine",
-                        "dynamicRangeFormats": ["HDR10", "SDR"]
+                        "dynamicRangeFormats": ["HDR10"]
                     }
                 },
                 "supportedStreamingTechnologies": ["DASH"]
